pages/reference-modules/4337/v0.3.0-1/operations/executeUserOpWithErrorString.mdx

```diff
@@ -0,0 +1,119 @@
+import { Tabs, Callout } from 'nextra/components'
+
+import LegacyCallout from '../../../../../components/callouts/LegacyCallout.mdx'
+import OnlySafeTxCallout from '../../../../../components/callouts/OnlySafeTxCallout.mdx'
+import ReentrancyCallout from '../../../../../components/callouts/ReentrancyCallout.mdx'
+import IrreversibilityCallout from '../../../../../components/callouts/IrreversibilityCallout.mdx'
+
+
+
+# `executeUserOpWithErrorString` (0x541d63c8)
+
+
+
+Executes a user operation provided by the entry point and returns error message on failure.
+
+
+
+
+Defined in [` Safe4337Module.sol`](https://github.com/safe-global/safe-modules/tree/4337/v0.3.0-1/modules/4337/contracts/Safe4337Module.sol#L185)
+
+
+## Usage
+
+{/* <!-- vale off --> */}
+
+<Tabs items={['example.sol']}>
+  <Tabs.Tab>
+    ```solidity
+    interface ISafe {
+        function executeUserOpWithErrorString(address to,uint256 value,bytes memory data,uint8 operation) external onlySupportedEntryPoint
+    }
+
+    contract Example {
+        function example() ... {
+            (ISafe safe).executeUserOpWithErrorString(
```

Comment:
Looks like the generated example code is generally off in all pages, so I will stop repeating myself.